doc/development/submitting/general.md

```diff
@@ -149,9 +149,9 @@ done manually, you can take an advantage of our [​CI](https://github.com/OSGeo
 
 Often there is more than one active release branch, if you are backporting, make
 sure you always backport to all the branches between "main" and the furthest
-branch you are backporting to. For example, let's say we have "main" and branches
-8.2 and 7.8, if you backport to 8.2, you should also consider to backport to 7.8.
-You can also choose to backport only to the closest branch, in our example 8.2.
+branch you are backporting to. For example, let's say we have "main" and release
+branches 3.5 and 3.6, if you backport to 3.5, you should backport to 3.6, too.
```

Comment:
Honestly, I find it confusing to first mention 3.5 and then 3.6.
Normally we *back*port, i.e. we go from higher to lower numbers.

Reply:
I did that because that's what the text before the example is describing, but I think I can modify it a bit to still keep it relevant to the text. Or maybe, the whole paragraph needs to be changed like putting the last sentence first. (The previous example was example of something else than what the text was describing.)

Reply:
The paragraph is now different and the example lists branches in the reverse order.